src/zjs_modules.c

```diff
@@ -123,8 +123,40 @@ static jerry_value_t native_require_handler(const jerry_value_t function_obj,
             return mod->instance;
         }
     }
+    ZJS_PRINT("Native module not found, searching for JavaScript module %s\n", module);
```

Comment:
We decided that since the native modules get printed during the build, the external JS ones should be too.  This only happens at build time so it won't add a bunch of noise to the output.